crates/db_queries/src/lib.rs

```diff
@@ -219,6 +221,22 @@ pub fn diesel_option_overwrite(opt: &Option<String>) -> Option<Option<String>> {
   }
 }
 
+pub fn diesel_option_overwrite_to_url(
+  opt: &Option<String>,
+) -> Result<Option<Option<lemmy_db_schema::Url>>, url::ParseError> {
+  Ok(match opt {
+    // An empty string is an erase
+    Some(unwrapped) => {
+      if !unwrapped.eq("") {
+        Some(Some(url::Url::parse(unwrapped)?.into()))
```

Comment:
Import `Url`.

Reply:
yeah, that's the reason, I guess this newtype is needed to workaround the orphan rule? maybe `DbUrl` would be most self-explanatory?

Reply:
That sounds good. You can change it as part of this PR if you dont mind.